developer/src/kmc-ldml/src/compiler/empty-compiler.ts

```diff
@@ -34,9 +34,12 @@ export class StrsCompiler extends EmptyCompiler {
     const strs = <KMXPlus.Strs>section;
 
     if (strs) {
+      let hadDenormalized = false;
       const badStringAnalyzer = new util.BadStringAnalyzer();
       const CONTAINS_MARKER_REGEX = new RegExp(LdmlKeyboardTypes.MarkerParser.ANY_MARKER_MATCH);
       for (let s of strs.allProcessedStrings.values()) {
+        // stop at the first denormalized string
+        hadDenormalized = hadDenormalized || util.isDenormalized(s);
```

Comment:
When we find the first denormalized string, stop checking for others.

Reply:
The only trick here is if the author would like to know about all the denormalized strings, they can't get a report of them. It might be better to report them all (once we have line numbers it will help) -- and then let the front end do the flood limiting that it already does on messages.

Reply:
will do